packages/e2e-tests/specs/editor/various/toolbar-roving-tabindex.test.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	createNewPost,
+	pressKeyWithModifier,
+	insertBlock,
+} from '@wordpress/e2e-test-utils';
+
+async function focusBlockToolbar() {
+	// Move the mouse to show the block toolbar
+	await page.mouse.move( 0, 0 );
```

Comment:
I don't know if that would be any better but there is a keyboard shortcut that moves focus to the block toolbar:

<img width="471" alt="Screen Shot 2020-06-17 at 14 30 20" src="https://user-images.githubusercontent.com/699132/84898157-182f4700-b0a7-11ea-93af-fef8e5478122.png">

I don't trust the stability of mouse interactions in e2e tests 😅

Reply:
Oh yeah! That reads better!